chalicelib/checks/higlass_checks.py

```diff
@@ -0,0 +1,799 @@
+from __future__ import print_function, unicode_literals
+from ..utils import (
+    check_function,
+    init_check_res,
+    action_function,
+    init_action_res
+)
+from dcicutils import ff_utils
+import requests
+import json
+import time
+
+def does_processed_file_have_auto_viewconf(hg_file):
+    """
+    See if the file has an auto generated higlass view config in its static contents.
+
+    Args:
+        hg_file(bool): A dict representing the file with static content.
+
+    Returns:
+        True if the file has an auto generated higlass view config, False otherwise
+    """
+    # - registered files will have a static_content item with description 'auto_generated_higlass_view_config'.
+
+    # it might be better to check the static_content.location instead...
```

Comment:
There could also eventually be some consideration for 'if it exists but is out of date' or similar somehow but not sure. Probably ok to come back to that Q later. Maybe could check date_modified or date_created of all processed_files and if any are more recent than when the HiGlassViewConfig was auto-generated, then it means that expset's viewconfig should be re-generated. Idk just thinking out...keyboard.